Source/WebCore/platform/ios/VideoFullscreenInterfaceAVKit.mm

```diff
@@ -450,7 +626,11 @@ - (void)exitFullScreenAnimated:(BOOL)animated completionHandler:(void (^)(BOOL s
 #define MY_NO_RETURN
 #endif
 
+#if HAVE(PIP_CONTROLLER)
+static const NSTimeInterval startPictureInPictureTimeInterval = 5.0;
+#else
 static const NSTimeInterval startPictureInPictureTimeInterval = 1.0;
```

Comment:
Could use the same value here, we have a radar to up the value for iOS because sometimes 1s wasn't enough.